site/docs/components/password_input.mdx

```diff
@@ -0,0 +1,256 @@
+---
+name: Password input
+menu: Components
+route: /components/password-input
+---
+
+import { Playground } from "docz";
+import { StatusLabel, PasswordInput, Button, IconEye, IconEyeCrossed } from "hds-react";
+
+import ColorBox from "../../src/components/ColorBox";
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+import Text from "../../src/components/Text";
+
+# Password input
+
+<StatusLabel type="alert">Draft</StatusLabel>
+<StatusLabel type="success" style={{marginLeft: 'var(--spacing-xs)'}}>Accessible</StatusLabel>
+
+<LargeParagraph>
+    A password field is a foundational input field component that the user can interact with and specifically input password data.
+</LargeParagraph>
+
+## Principles
+- A label should always be provided with password fields.
+- If possible, allow the password input to reflect the strength of the password entered.
```

Comment:
... when the password field component is used for creating a new password.

Reply:
Maybe split this documentation into 2 parts depending on use case:
1. entering current password for login
2. creating a new password